package.json

```diff
@@ -52,9 +52,8 @@
   },
   "scripts": {
     "browserstack": "sh build/run-browserstack.sh",
-    "ci": "grunt test",
-    "ci-include-lint": "grunt eslint && grunt test",
-    "test": "grunt",
+    "build": "grunt build",
+    "test": "grunt test",
```

Comment:
jQuery doesn't build on `grunt test` but it does on `npm test`:
https://github.com/jquery/jquery/blob/master/package.json#L63

I think it makes sense as it allows for the standard `npm install && npm test` (or `npm it` for short) workflow without diving into the project internals and without relying on a deprecated hook.